plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Tax.php

```diff
@@ -129,6 +129,7 @@ public function get_additional_data() {
 		return array(
 			'avalara_activated'         => PluginsHelper::is_plugin_active( 'woocommerce-avatax' ),
 			'tax_jar_activated'         => class_exists( 'WC_Taxjar' ),
+			'woocommerce_tax_activated' => PluginsHelper::is_plugin_active( 'woocommerce-tax' ),
```

Comment:
Informs [this functionality](https://github.com/woocommerce/woocommerce/pull/46929/files#r1629385636) whether the WC Tax plugin is installed.